### DISCUSSION TITLE
Configuration Overrides

### DISCUSSION THREAD

**Discussion by darac** [tiredofit/docker-rspamd#9] [Q&A]
Hello,

I'm looking to migrate my rspamd instance from the `a16bitsysop/docker-rspamd` image to this image, but I notice that there are some blank sections in the README here. Specifically, there doesn't seem to be any documentation about how to configure this image.

With the `a16bitsysop/docker-rspamd`, I have two volumes mapped:
* One maps to `/var/lib/rspamd`, and holds rspamd's persistent data (mostly hyperscan files)
* A second one maps to `/etc/rspamd/override.d`, and provides me somewhere to store `*.inc` and `*.conf` files.

Is it possible to configure this image in the same manner?

**Comment by tiredofit** [comment]
Have a look at the example docker-compose - I use the folder `/data` which is essentially the /var/lib/rspamd persistent data, however if you are using dkim I also store keys in a subfolder in there. You should be able to map the override.d folder as well. For the most part the container auto configures itself and then environment variables can be changed. I have yet to require any override files. Have a look at install/assets/defaults/10-rspamd for all of the environment variables.
